src/Uno.Extensions.Reactive/Core/Axes/MessageAxis.T.cs

```diff
@@ -35,7 +35,7 @@ internal MessageAxis(string name, Aggregator aggregate)
 	/// <param name="value">The metadata to encapsulate..</param>
 	/// <returns>The raw axis value.</returns>
 	[Pure]
-	public MessageAxisValue ToMessageValue(T? value)
+	public MessageAxisValue ToMessageValue(T value)
```

Comment:
The naming for the From and To methods might be better as ToAxisValue and FromAxisValue